### PR TITLE
Add integration file with metadata for energy

### PR DESCRIPTION
**Pull request by ludeeus**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/energy.markdown [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (ludeeus) [inline source/_integrations/energy.markdown:1]: Redirect does not seem to work https://deploy-preview-20232--home-assistant-docs.netlify.app/integrations/energy/

Comment (frenck) [reply source/_integrations/energy.markdown:1]: Correct, you need to force the redirect in this case (files take precedence over redirects)

Comment (frenck) [reply source/_integrations/energy.markdown:1]: Hmm that seems only possible using a netlify configuration file

https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Comment (frenck) [reply source/_integrations/energy.markdown:1]: Hmm that seems only possible using a netlify configuration file

https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Comment (ludeeus) [reply source/_integrations/energy.markdown:1]: `forced` is only for that file, `301!`  should work <https://www.netlify.com/blog/2020/04/07/creating-better-more-predictable-redirect-rules-for-spas/>

Comment (ludeeus) [reply source/_integrations/energy.markdown:1]: Working as expected now :+1:

Comment (frenck) [reply source/_integrations/energy.markdown:1]: TIL! Nice! That should be in their docs 😬

**Review by frenck — APPROVED** [review]

**Pull request merged**